frontend/styles/global/_global.scss

```diff
@@ -97,7 +97,6 @@ form,
 
   .button-wrap {
     margin: 0;
-    margin-top: $pad-small; // 16px gap between form fields + this additional 8px = style-guide specified 24px between form submit and above field
```

Comment:
This moves the buttons under every form in the app a bit closer to the form above them - since I don't see this in the design system (could be missing it), wanted to double-checking we want to do this?
<img width="814" height="790" alt="Screenshot 2025-09-29 at 4 45 55 PM" src="https://github.com/user-attachments/assets/408e1dcb-dc20-4ad5-ba84-757c50d754d9" />